vendor/engines/split_accounts/app/models/split_accounts/split_account.rb

```diff
@@ -18,7 +18,9 @@ def valid_percent_total
     end
 
     def percent_total
-      splits.reduce(0) { |sum, split| sum + split.percent }
+      splits.reduce(0) do |sum, split|
+        split.percent.present? ? sum + split.percent : sum
```

Comment:
What about `splits.reduce(0) { |sum, split| sum + split.percent.to_i }`? I think that would turn `nil`s into 0. Just a thought.